### PR TITLE
Add extra http headers in code and test description in README

### PR DESCRIPTION
**Pull request by rstarmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ Optune servo driver for Apache Benchmark
 Supported environment variables:
 
 * `AB_TEST_URL` - URL to measure
+* `AB_EXTRA_HEADERS` - comman separated key:value headers: e.g.:
+```host:hostname.local,X-AUTH-TOKEN:aTokenForAuthentication```
```
Comment (pavel-georgiev) [inline README.md]: How are headers with ',' in them handled?

File: README.md [hunk 2]
```diff
@@ -21,4 +23,42 @@ Note: a test URL must be defined either in the environment variable or in the `l
...
+{}
+EOF
+```
+Pass multiple Header parameters to the measure command as JSON (_note_: multiple headers need to be in a proper JSON list):
+```
+export AB_TEST_URL=http://localhost:8090/?busy=5
+measure front <<EOF
+{"control":"load":{"extra_headers":["host:back.example.com","x-auth-token:ThisShouldBeAnAuthToken"]}}}
+EOF
+```
```
Comment (pavel-georgiev) [inline README.md:66]: 1. How is the success of the test verified? 
2. Are headers supported both as env var and as part of the input?

File: measure [hunk 3]
```diff
@@ -17,12 +17,13 @@ HAS_CANCEL=True
...
-    't_warmup'   :  30,         # warmup time in seconds:  0 => no warmup
-    'user'       : '',
-    'password'   : '',
+    'n_threads'     : 10,
+    'n_requests'    : 10000000,   # request limit
+    't_limit'       : 180,         # time limit in seconds:   0 => no time limit
+    't_warmup'      :  30,         # warmup time in seconds:  0 => no warmup
+    'user'          : '',
+    'password'      : '',
+    'extra_headers' : [],
```
Comment (pnickolov) [inline measure:26]: why do we call them "extra" headers; I would propose calling it simply "headers" everywhere (env var and load parameter)

File: README.md [hunk 4]
```diff
@@ -21,4 +23,42 @@ Note: a test URL must be defined either in the environment variable or in the `l
...
+
+## Testing the driver
+
+Launch a simple test app that does some processing (e.g. docker run --rm -d -p 8080:8080 opsani/co-http)
+
+Run a test using the defaults from the command line:
+
+```
+export AB_TEST_URL=http://localhost:8080/?busy=5
+measure test <<EOF
```
Comment (pnickolov) [inline README.md:38]: shouldn't this be ./measure instead of measure (here and below)

File: README.md (hunk 2 above)
Comment (pavel-georgiev) [reply README.md:66]: 3. README should specify that extra_headers is one of the supported input parameters, same as `user ` and `password`.

**Pull request merged**